docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2266,6 +2267,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/core/capabilitiesUrl`|null|
 |`fhirServer/core/externalBaseUrl`|null|
 |`fhirServer/core/ifNoneMatchReturnsNotModified`|false|
+|`fhirServer/core/defaultFhirVersion`|null|
```

Comment:
Shouldn't we specify a default for this?  Otherwise, what is the default when the user doesn't specify a fhirVersion parameter on the header.

Reply:
Looks like in FHIRVersionRequestFilter.java, the default is 4.0 if nothing is set here, so I think that should be reflected here.

Reply:
ok.  i think i went back and forth on "what should be the default".
and its actually a two-part question:
1. what is the default if no config value is provided
2. what should we put in our default fhir-server-config

Reply:
for now i'll just update the user guide to 4.0 as suggested since, as you say, thats whats in the code

Reply:
done